BaseListener/Boundary/EventData.cs

```diff
@@ -0,0 +1,9 @@
+namespace BaseListener.Boundary
+{
+    public class EventData
```

Comment:
is this only useful for the activity listener?

Reply:
Maybe, but it will be included on the raised event anyway.